oscarapi/tests/unit/testcheckout.py

```diff
@@ -35,7 +35,7 @@ class CheckoutTest(APITest):
     def _get_common_payload(self, basket_url):
         return {
             "basket": basket_url,
-            "guest_email": "henk@example.com",
+            "email": "henk@example.com",
```

Comment:
Hmm, I see now that you changed the payload for the checkout serializer in the tests.

The CheckoutSerializer and OrderSerializer are not the same, so I think you should not rename this field without updating the checkout serializer and logic too. The CheckoutSerializer uses the guest_email property to store orders for anonymous users. 

Please discuss with @specialunderwear if you need to update the checkout api too.